scripts/create_game.py

```diff
@@ -0,0 +1,90 @@
+# SPDX-FileCopyrightText: 2022, Texas Advanced Computing Center
+# SPDX-License-Identifier: BSD-3-Clause
+"""Create Diplomacy games programmatically."""
+# Based on https://github.com/SHADE-AI/diplomacy-playground/blob/989ec7be748257324a22132e632ac4927b8cb6c2/scripts/create_game.py
+import argparse
+import asyncio
+import json
+from typing import Optional, Sequence
+
+from diplomacy.client.connection import connect
+
+DEFAULT_RULES = ("REAL_TIME", "POWER_CHOICE")
+
+
+async def create_game(
+    game_id: str,
+    rules: Sequence[str] = DEFAULT_RULES,
+    deadline: int = 0,
+    n_controls: int = 7,
+    password: Optional[str] = None,
+    hostname: str = "localhost",
+    port: int = 8432,
+) -> None:
+    """Creates a game on the Diplomacy server"""
+    connection = await connect(hostname, port)
+    channel = await connection.authenticate("random_user", "password")
```

Comment:
Random user might be best to make a meaningful name if we submit a game for TACC review

Reply:
I agree. After the problem today, I made a note to have a better default for this.